addons/website_project_issue/controllers/main.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import werkzeug
+
+from openerp import http
+from openerp.http import request
+
+from openerp.addons.website_portal.controllers.main import website_account
+
+class WebsiteAccount(website_account):
+    @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
+    def account(self):
+        response = super(WebsiteAccount, self).account()
+        project_issues = request.env['project.issue'].search([])
+        response.qcontext.update({
+            'issues': project_issues
+        })
+        return response
+
+class website_project_issues(http.Controller):
+    @http.route(['/my/issues/<model("project.issue"):issue>'], type='http', auth="user", website=True)
+    def issues_followup(self, issue=None):
+        has_timesheet_enabled = issue._fields.get('timesheet_ids')
+        note_subtype_id = request.env.ref('mail.mt_note').id
+        # Filtering chatter messages as we do not want to display internal notes
+        messages = issue.message_ids.filtered(lambda message: message.subtype_id.id != note_subtype_id and message.body)
```

Comment:
can you see this record rule : https://github.com/odoo/odoo/blob/master/addons/mail/security/mail_security.xml#L41 ?

so no need to filtered internal notes!